tests/clickhouse-test

```diff
@@ -2269,11 +2269,11 @@ def reportLogStats(args):
               'Attempt to read after eof', 'String size is too big ({}), maximum: {}'
         ) AS known_short_messages
         SELECT count() AS c, message_format_string, substr(any(message), 1, 120),
-            min(if(length(regexpExtract(message, '(.*)\\([A-Z0-9_]+\\)')) as pref > 0, pref, length(message)) - 26 AS length_without_exception_boilerplate) AS min_length_without_exception_boilerplate
+            min(if(notEmpty(regexpExtract(message, '(.*)\\([A-Z0-9_]+\\)') as prefix), prefix, length(message)) - 26 AS length_without_exception_boilerplate) AS min_length_without_exception_boilerplate
```

Comment:
This break the check:
```
Failed to get stats about log messages: Code: 500. Code: 386. DB::Exception: There is no supertype for types String, UInt64 because some of them are String/FixedString and some of them are not: While processing if(notEmpty(regexpExtract(message, '(.*)\\([A-Z0-9_]+\\)') AS prefix), prefix, length(message)). (NO_COMMON_TYPE) (version 23.12.1.1000 (official build))
```
https://s3.amazonaws.com/clickhouse-test-reports/0/7335707f9a654808d151a68a7b0d99e09d49f9c8/stateless_tests__release__s3_storage__[1_2].html

Reply:
What did the change try to fix?